core/input/input.cpp

```diff
@@ -135,6 +135,8 @@ void Input::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_use_accumulated_input", "enable"), &Input::set_use_accumulated_input);
 	ClassDB::bind_method(D_METHOD("flush_buffered_events"), &Input::flush_buffered_events);
 
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "mouse_mode"), "set_mouse_mode", "get_mouse_mode");
```

Comment:
Maybe worth giving it a `PROPERTY_HINT_ENUM`, so values display correctly in places like VisualScript and Inspector.